x-pack/plugins/cases/common/api/connectors/index.ts

```diff
@@ -5,139 +5,11 @@
  * 2.0.
  */
 
-import * as rt from 'io-ts';
```

Comment:
All of this except the get connectors API schema was moved to the `connector.ts` file to help avoid circular dependencies.